lib/l10n/app_de.arb

```
@@ -190,5 +190,14 @@
   "signerErrorRequestInProgress": "Eine andere Anfrage wird gerade bearbeitet. Warte bitte.",
   "signerErrorNoActivity": "Signer konnte nicht gestartet werden. Bitte versuch es erneut.",
   "signerErrorLaunchError": "Signer-App konnte nicht gestartet werden.",
-  "signerErrorUnknown": "Ein Fehler ist beim Signer aufgetreten. Bitte versuch es erneut."
+  "signerErrorUnknown": "Ein Fehler ist beim Signer aufgetreten. Bitte versuch es erneut.",
+  "carouselPrivacyTitle": "Privatsphäre und Sicherheit",
+  "carouselPrivacyDescription": "Halte deine Gespräche privat. Selbst bei einem Datenleck bleiben deine Nachrichten sicher.",
+  "carouselIdentityTitle": "Wähle deine Identität",
+  "carouselIdentityDescription": "Chatte ohne deine Telefonnummer oder E-Mail preiszugeben. Wähle deine Identität: echter Name, Pseudonym oder anonym.",
+  "carouselDecentralizedTitle": "Dezentralisiert und genehmigungsfrei",
+  "carouselDecentralizedDescription": "Keine zentrale Instanz kontrolliert deine Kommunikation – keine Genehmigungen nötig, keine Zensur möglich."
+,
```

RuboCop (1.84.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)